packages/scene-composer/src/components/toolbars/common/ToolbarItem.tsx

```diff
@@ -6,15 +6,27 @@ import { ItemContainer } from './ItemContainer';
 import { CornerAdornment, ToolbarItemMenu } from './styledComponents';
 import { ToolbarItemOptions, ToolbarItemOrientation, ToolbarItemType, ToolbarMenuPosition } from './types';
 
+interface MenuItemsContainterProps {
+  maxHeight?: string;
+  children?: React.ReactNode;
+}
+
+const MenuItemsContainter = ({ maxHeight, children }: MenuItemsContainterProps) => {
+  return <div style={{ overflowY: 'auto', maxHeight: maxHeight }}>{children}</div>;
+};
+
 interface ToolbarItemProps<T extends ToolbarItemOptions> {
   items: T[];
   type: ToolbarItemType;
   initialSelectedItem?: T;
   onSelect?: (item: T) => void;
   orientation?: ToolbarItemOrientation;
+  isVertical?: boolean;
   menuPosition?: ToolbarMenuPosition;
   // Only used by the first root menu item
   menuHeight?: string;
+  // For the box containing the menu items
```

Comment:
NIT:  No need of this comment. Variable name below is self explanatory.

Reply:
Removed